docs/source/search/searching_edsc_gui.ipynb

```diff
@@ -13,7 +13,7 @@
    "source": [
     "The Earthdata Search Client (EDSC) allows users to search, preview, download, and access EOSDIS Earth observation data. The MAAP has its own separate instance of NASA's EDSC which is located at [search.maap-project.org](http://search.maap-project.org). This MAAP ESDC provides a graphical user interface (GUI) for the MAAP's Common Metadata Repository (CMR) in order to make the process of discovering data easy for MAAP users. \n",
     "\n",
-    "![Earthdata Search Client GUI](../_static/edsc_gui.png)\n",
+    "![Earthdata Search Client GUI](../_static/edsc_gui.PNG)\n",
```

Comment:
nit: `git mv` to make the file name lowercase instead, so all the ext are consistent case.